modules/gfac/gfac-core/src/main/java/org/apache/airavata/gfac/core/context/ProcessContext.java

```diff
@@ -681,7 +687,7 @@ public String getQualityOfService() {
 	public String getQueueName() {
 		if (isUseUserCRPref() &&
```

Comment:
Also, I don't think this logic is correct. I think processModel.getProcessResourceSchedule() should be preferred first and if it doesn't specify a queue then it should fall back to the preferences.

Reply:
I kept it as it is as it was in the initial implementation. Will bring it into top. Thanks for pointing out.